BinaryEncoding.md

```diff
@@ -59,6 +59,32 @@ A single-byte unsigned integer indicating a [value type](AstSemantics.md#types).
 * `3` indicating type `f32` 
 * `4` indicating type `f64`
 
+### definition_kind
+A single-byte unsigned integer indicating the kind of definition being imported or defined:
+* `0` indicating a `Function` [import](Modules.md#imports) or [definition](Modules.md#function-and-code-sections)
+* `1` indicating a `Table` [import](Modules.md#imports) or [definition](Modules.md#table-section)
+* `2` indicating a `Memory` [import](Modules.md#imports) or [definition](Modules.md#linear-memory-section)
+* `3` indicating a `Global` [import](Modules.md#imports) or [definition](Modules.md#global-section)
+
+### resizable_definition
+A packed tuple that describes the import or definition of a resizable 
+[table](AstSemantics.md#table) or [memory](AstSemantics.md#resizing):
+
+| Field | Type | Description |
+| ----- |  ----- | ----- |
+| flags | `varuint32` | described below |
+| initial | `varuint32` | initial length (in units of table elements or wasm pages) |
+| maximum | `varuint32`? | only present if specified by `flags` |
+
+The `varuint32` "flags" field assigns the following meaning to the bits:
+* `0x1` : this is a *default* [table](AstSemantics.md#table)/[memory](AstSemantics.md#linear-memory)
+          (in the MVP, *every* memory/table import/definition must be flagged as default)
+* `0x2` : indicates that the memory/table has a specified maximum
+
+### init_expr
+The encoding of an [initializer expression](Modules.md#initializer-expression)
+is simply the encoding of the operator (`*.const` or `get_global`) followed by
+its immediate (constant value or global index).
```

Comment:
This may not be future proof. If we want to allow more operators in the future then we probably need to delimit the operator sequence somehow, e.g. with an `end` opcode.